playground/AspireEventHub/EventHubsConsumer/Processor.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Processor;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace EventHubsConsumer;
+
+internal sealed class Processor(EventProcessorClient client, ILogger<Consumer> logger) : BackgroundService
+{
+    public override async Task StartAsync(CancellationToken cancellationToken)
+    {
+        logger.LogInformation("Starting processor...");
+
+        client.ProcessEventAsync += ClientOnProcessEventAsync;
+        
+        Task ClientOnProcessEventAsync(ProcessEventArgs arg)
+        {
+            logger.LogInformation(arg.Data.EventBody.ToString());
+            return Task.CompletedTask;
+        }
+
+        client.ProcessErrorAsync += args =>
+        {
+            logger.LogError(args.Exception, "Error processing message: {Error}", args.Exception.Message);
+            return Task.CompletedTask;
+        };
+
+        await client.StartProcessingAsync(cancellationToken);
```

Comment:
We should include comments that discuss the threading model for the processor and guidance for partition ownership.  This pattern is great for a simple sample but goes against how the processor is intended to be used for the majority of production scenarios.

Too often we see developers attempt to handle too many partitions on a single machine and see the expected poor performance as a result.

Reply:
Good call. I'll link to the guidance and insert a pertinent quote as a comment.

Reply:
Done.